graspy/embed/embed.py

```diff
@@ -78,17 +105,16 @@ def fit(self, graph):
         # import graph(s) to an adjacency matrix using import_graph function
         # here
 
-        return self.lpm
+        return self
 
     def _fit_transform(self, graph):
         "Fits the model and returns the estimated latent positions"
-        try:
```

Comment:
If calling fit_transform with new data than the model was fit on, we want fit_transform to rerun fit regardless. Is that the purpose of removing the _check_is_fit? just making sure

Reply:
yea i ran into instances where i was calling fit_transform on different data, and realized it wasn't doing anything cause of these lines. you could instantiate more than one class, but seems cumbersome? This is also how fit_transform works in sklearn when I checked.

Reply:
Yes i agree with this, just wanted to make sure it's the behavior we want